### PR TITLE
Contributing under GSSOC'24. Added a new game 'Color Blink Challenge'

### PR DESCRIPTION
**Pull request by rugved0102**
## PR Description 📜

The Color Blink Challenge is a fun and interactive game where you need to count how many times a specific color appears within 30 seconds. The color to count is selected randomly at the start of each round. Test your focus and speed, and try to achieve the highest score possible!
Fixes # <your_issue_number>

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in GameZone's README.md
- [x] I have added README.md in my Game folder 

<hr>

## Add your screenshots(Optional) 📸
image

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @rugved0102 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

File: README.md [hunk 1]
```diff
@@ -915,6 +915,7 @@ This repository also provides one such platforms where contributers come over an
 | [Drum_kit] (https://github.com/kunjgit/GameZone/tree/main/Games/Drum_kit)|
 | [Prime-No-Finder-Game] (https://github.com/kunjgit/GameZone/tree/main/Games/Prime-No-Finder-Game)|  
 | [Synonym-Typing-Game] (https://github.com/kunjgit/GameZone/tree/main/Games/Synonym-Typing-Game)|
+| [Color Blink Challenge] (https://github.com/kunjgit/GameZone/tree/main/Games/Color Blink Challenge)|
```
Comment (ishita-43) [inline README.md:918]: Follow  naming convention while naming game folder and respectively change the game folder after that here

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Make following change

File: README.md (hunk 1 above)
Comment (rugved0102) [reply README.md]: @ishita-43  can you please confirm ? I have done the changes.

**Review by ishita-43 — APPROVED** [review]
Ready to merge

**Comment by rugved0102** [comment]
@kunjgit can you please merge my pr ?

**Comment by kunjgit** [comment]
Thank you  @rugved0102 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**